build.sbt

```diff
@@ -7,7 +7,7 @@ val commonSettings = Seq(
   organization := "com.rms.miu",
 
   scalaVersion := "2.12.10",
-  crossScalaVersions := Seq("2.11.12", "2.12.10","2.13.1"),
+  crossScalaVersions := Seq("2.12.10","2.13.1"),
```

Comment:
To clarify - are we dropping 2.11 because of missing ScalaTest or what exactly?

Reply:
Cats 2.1 dropped scala 2.11 support.

Reply:
👍 good to know.